testing/minikube/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor:
+# Maintainer: Magicloud <magiclouds@gmail.com>
+pkgname=minikube
+pkgver=0.35.0
+pkgrel=0
+pkgdesc="Minikube is a tool that makes it easy to run Kubernetes locally"
+url="https://github.com/kubernetes/minikube"
+arch="all"
+license="Apache"
```

Comment:
The license should follow the SPDX naming conversion - https://spdx.org/licenses/

Reply:
OK.